tests/sampling/test_jax.py

```diff
@@ -536,3 +542,31 @@ def test_dirichlet_multinomial_dims():
     frozen_dm = freeze_dims_and_data(m)["dm"]
     dm_draws = pm.draw(frozen_dm, mode="JAX")
     np.testing.assert_equal(dm_draws, np.eye(3) * 5)
+
+
+@pytest.mark.parametrize("method", ["advi", "fullrank_advi"])
+def test_vi_sampling_jax(method):
+    with pm.Model() as model:
+        x = pm.Normal("x")
+        pm.fit(10, method=method, fn_kwargs=dict(mode="JAX"))
+
+
+@pytest.mark.xfail(
+    reason="""
+During equilibrium rewriter this error happens. Probably one of the routines in SVGD is problematic.
+
+TypeError: The broadcast pattern of the output of scan
+(Matrix(float64, shape=(?, 1))) is inconsistent with the one provided in `output_info`
+(Vector(float64, shape=(?,))). The output on axis 0 is `True`, but it is `False` on axis
```

Comment:
This is actually something wrong, if the number of dimensions of s recurring output is different from the initial state. The difference between None and 1 is more annoying but this one looks like an error.